ocppj/ocppj.go

```diff
@@ -359,7 +371,7 @@ func (endpoint *Endpoint) ParseMessage(arr []interface{}, pendingRequestState Cl
 		}
 		err = Validate.Struct(callResult)
 		if err != nil {
-			return nil, errorFromValidation(err.(validator.ValidationErrors), uniqueId)
+			return nil, errorFromValidation(err.(validator.ValidationErrors), uniqueId, request.GetFeatureName()+"Confirmation")
```

Comment:
I would change this to `Response`. Or, not pass this info at all (just pass the feature name). Imho that bit of additional info is not a game-changer, since the endpoint receiving this error can infer everything from:
- message ID
- content of the error

Reply:
True! The field name includes `Call` or `CallResult`. I've included just the feature name.